user_manual_de/pim/sync_thunderbird.rst

```diff
@@ -1,29 +1,45 @@
-Thunderbird - Adressbuch synchronisieren
-========================================
+===============================
+Synchronisation mit Thunderbird
+==============================
```

Comment:
```suggestion
===============================
```

Reply:
@vasyugan it is still missing one char length :)

Reply:
Hm, it seems the pull request is still open. I have accepted all your proposed changes. So what is holding it up?

Reply:
The one above i still not fixed :)